internal/frontend/auth/README.md

```
@@ -0,0 +1,171 @@
+# Authentication Frontend
+
+The `internal/frontend/auth/` package contains the login page handler and frontend content.
+The frontend is a standalone React + PatternFly 6 project that provides the OIDC login
+and device authorization pages. The assets are embedded into the Go binary at compile time.
+
+## Directory Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)